### PR TITLE
Bigdecimal support

### PR DESCRIPTION
**Pull request by Pscheidl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,6 @@ Cargo.lock
 # code coverage
 /codecov.json
 /lcov.info
+
+#IDEs
+.idea/
```
Comment (robjtede) [inline .gitignore:13]: echo '.idea/' >> .git/info/exclude

```suggestion
```

File: confik/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Implement `Configuration` for [`bigdecimal::BigDecimal`](https://docs.rs/bigdecimal/0.4.3/bigdecimal/struct.BigDecimal.html).
```
Comment (robjtede) [inline confik/CHANGELOG.md:5]:
```suggestion
- Implement `Configuration` for [`bigdecimal::BigDecimal`](https://docs.rs/bigdecimal/0.4/bigdecimal/struct.BigDecimal.html).
```

File: confik/Cargo.toml [hunk 3]
```diff
@@ -16,8 +16,8 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [package.metadata.cargo-machete]
 ignored = [
-  "humantime_serde", # used in serde macro attributes
-  "serde_with",      # used in doctests
+    "humantime_serde", # used in serde macro attributes
+    "serde_with", # used in doctests
```
Comment (robjtede) [inline confik/Cargo.toml:20]: please revert

File: confik/Cargo.toml [hunk 4]
```diff
@@ -50,6 +51,7 @@ envious = { version = "0.2", optional = true }
 serde_json = { version = "1", optional = true }
 toml = { version = "0.8", optional = true, default-features = false, features = ["parse"] }
 
+bigdecimal = { version = "0.4.3", optional = true, features = ["serde"] }
```
Comment (robjtede) [inline confik/Cargo.toml:54]:
```suggestion
bigdecimal = { version = "0.4", optional = true, features = ["serde"] }
```

**Review by robjtede — APPROVED** [review]
also please add mention to this section of docs: <https://docs.rs/confik/latest/confik/#handling-foreign-types>

File: confik/Cargo.toml (hunk 3 above)
Comment (Pscheidl) [reply confik/Cargo.toml]: Reverted, unintentional.

**Review by robjtede — DISMISSED** [review]

**Comment by Pscheidl** [comment]
Thank you for the review, I've addressed all comments and added a mention to the `handling foreign types` section.

**Review by robjtede — APPROVED** [review]

**Pull request merged**